testing/snapshot_test.ts

```diff
@@ -491,3 +497,55 @@ Deno.test("Snapshot Test - Regression #2144", async (t) => {
 Deno.test("Snapshot Test - Empty #2245", async (t) => {
   await assertSnapshot(t, "", { serializer: (x) => x });
 });
+
+Deno.test("SnapshotTest - createAssertSnapshot", async (t) => {
+  const assertMonochromeSnapshot = createAssertSnapshot<string>({
+    serializer: stripColor,
+  });
+
+  await t.step("No Options", async (t) => {
+    await assertMonochromeSnapshot(
+      t,
+      "\x1b[32mThis green text has had it's colours stripped\x1b[39m",
+    );
+  });
+
+  await t.step("Options Object", async (t) => {
+    await assertMonochromeSnapshot(
+      t,
+      "\x1b[32mThis green text has had it's colours stripped\x1b[39m",
+      {
+        name:
+          "SnapshotTest - createAssertSnapshot - Options Object - Custom Name",
+      },
+    );
+  });
+
+  await t.step("Message", async (t) => {
+    const assertMissingSnapshot = createAssertSnapshot<string>({
+      mode: "assert",
+      name: "[MISSING SNAPSHOT]",
+    });
+
+    const err = await assertRejects<AssertionError>(async () => {
```

Comment:
Let's use `assertRejects(fn, AssertionError)` call, that will give correct type to `err`